applications/salk-portal/src/components/ExperimentSidebar.jsx

```diff
@@ -125,6 +174,16 @@ const ExperimentSidebar = () => {
   };
 
   const sidebarClass = `${classes.sidebar} scrollbar ${shrink ? `${classes.shrink}` : ``}`;
+  const populationColours = ['green', 'sky', 'purple', 'brown', 'blue'];
+
+  const PopulationLabel = ({labelText}) => {
+    return (
+      <div className='sidebar-label'>
+        <div className={`sidebar-dot-${populationColours[Math.floor(Math.random() * populationColours.length)]}`}></div>
```

Comment:
if you do this (random choice) and you don't remove the used colors from the color list then 2 populations could get the same color, I think that's not what we want but please check this with @vidhya-metacell

Reply:
@paularah i would suggest to use array index at present for the color selection, which would avoid repetitive color. It's only a static UI right now, so info about color and how to relate it on UI would come later on.
I would keep a color array fetching variables from variables/theme file and then use that to pass the background Color info